### PR TITLE
Update fr.po

### PR DESCRIPTION
**Pull request by Surfoo**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -2224,7 +2224,7 @@ msgstr "En cours de téléchargement"
 
 #: ../glade/simple_search.ui.h:9 ../glade/update_monitor.ui.h:4
 msgid "Cancel _All"
-msgstr "_Annuler le tout"
+msgstr "Tout _Annuler"
```
Comment (fitojb) [inline po/fr.po]: I don’t think it’s correct in French to copy English’s “title case” style.

Comment (Surfoo) [inline po/fr.po]: I'm french, yes it's correct :)

Comment (fitojb) [inline po/fr.po]: _Annuler_ is a verb and not a proper noun; a capital letter doesn’t correspond there. See https://fr.wikipedia.org/wiki/Usage_des_majuscules_en_fran%C3%A7ais

**Comment by Surfoo** [comment]
Yes indeed, It's updated!

**Pull request merged**